service/pixelated/account_recovery.py

```diff
@@ -13,23 +13,51 @@
 #
 # You should have received a copy of the GNU Affero General Public License
 # along with Pixelated. If not, see <http://www.gnu.org/licenses/>.
+
 from twisted.internet.defer import inlineCallbacks, returnValue
 from twisted.logger import Logger
+from twisted.mail import smtp
+
+from email.mime.text import MIMEText
+
 
 log = Logger()
 
 
 class AccountRecovery(object):
-    def __init__(self, session, soledad):
+    def __init__(self, session, soledad, smtp_config, backup_email):
         self._bonafide_session = session
         self._soledad = soledad
+        self._smtp_config = smtp_config
+        self._backup_email = backup_email
 
     @inlineCallbacks
     def update_recovery_code(self):
         try:
             code = self._soledad.create_recovery_code()
             response = yield self._bonafide_session.update_recovery_code(code)
+            yield self._send_mail(code, self._backup_email)
+
             returnValue(response)
+
         except Exception as e:
             log.warn('Something went wrong when trying to save the recovery code')
             raise
+
+    @inlineCallbacks
+    def _send_mail(self, code, backup_email):
+        msg = MIMEText('Your code %s' % code)
+        msg['Subject'] = 'Recovery Code'
+        msg['From'] = 'team@pixelated-project.org'
+        msg['To'] = backup_email
```

Comment:
Should we have all of these in a file like we do with the welcome mail? Or maybe have the recovery code email as an object in itself? I don't know, just thinking of how to keep this clean, but I think this is something for the future.